pkg/cvo/cvo.go

```diff
@@ -269,6 +271,24 @@ func (optr *Operator) InitializeFromPayload(restConfig *rest.Config, burstRestCo
 	return nil
 }
 
+func (optr *Operator) ownerReferenceModifier(object metav1.Object) {
+	var found bool
+	for _, ownerRef := range object.GetOwnerReferences() {
+		if ownerRef.Kind == "ClusterVersion" && ownerRef.Name == optr.name {
```

Comment:
We should be careful adding CVO ownerReference to an object that already has an owner.  If the object has an owner because it was part of a cascading delete set-up then adding CVO would interfere with that (See discussions in the [Spike](https://issues.redhat.com/browse/OTA-403)).

I thought it would be worthwhile to log those objects that already had an owner to do some spot checking. May be a non-issue but not sure.

Reply:
I've updated the logic to only set the CV owner reference when there is no existing owner reference which is not a CV resource.